### DISCUSSION TITLE
How use Remote SSH

### DISCUSSION THREAD

**Discussion by guilhermeaddr** [coder/code-server#4462] [Q&A] [answered]
Hello everyone, I just installed the code-server using the linuxserver docker image and managed to install the "Remote SSH" extension, but when I try to connect to a remote server nothing happens. It seems to me that when I start the code-serve, it already is connected to its own remote environment "nas:8443". Anyone have a solution, please?

![imagem_2021-11-07_101346](https://user-images.githubusercontent.com/93304166/140646442-9beb9991-af87-4f25-830e-c9c6acff21f9.png)

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
You can't use Remote SSH on code-server since code-server itself is (kinda) a Remote SSH session. Instead, if you gain access to the system, you can:

- Get a DE like Xfce
- Get VNC server/Xrdp and use it to connect to your deployment's GUI.

Then you can install VS Code, get the Remote SSH extension and use it.

**Reply by guilhermeaddr** [reply]
Thanks friend for the answer, I already found a way around this. I run my ssh environments (node, python) inside Docker containers, so I'm finished creating images with the codeserver installed, thanks a lot.

**Reply by guilhermeaddr** [reply]
Done, now I can access the environment with vscode installed on my pc using "Remote SSH" and also by code-server via browser. 
https://github.com/guidevops/docker-workspace

**Reply by maestrocoder** [reply]
Kudos guidevops!  Are you still using the docker-workspace solution?
